openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -170,11 +170,9 @@ Credentials can therefore vary in the following dimensions:
 - Credential Format
 - Cryptographic Data
 
-In the context of a single request, all issued Credentials MUST share the same Credential Format and Credential Dataset. However, they MAY contain different Cryptographic Data, such as being bound to different cryptographic keys (e.g., to ensure unlinkability between the Credentials).
+In the context of a single request, the batch of issued Credentials MUST share the same Credential Format and Credential Dataset. However, they MAY contain different Cryptographic Data, such as being bound to different cryptographic keys (e.g., to ensure unlinkability between the Credentials).
```

Comment:
Again not a change introduced by this PR but I think we should consider making the MAY here even stronger, for example when would we want to allow multiple instances of a credential to be issued to the same keys?

Reply:
@tplooker pls re-review